src/components/Album.jsx

```diff
@@ -0,0 +1,26 @@
+import { Header } from './Header.jsx'
+import { ArtistName } from './ArtistName.jsx'
+import { CoverImage } from './CoverImage.jsx'
+import { AlbumName } from './AlbumName.jsx'
+
+export const Album = ({ dataProp }) => {
+    const albumData = dataProp.albums.items
+    return (
+
+        <div className="album">
+            <Header />
+            {albumData.map((album) => {
```

Comment:
the return method here needs to be stored in a variable to be able to invoke it later in your Album function